ai_genomics/getters/data_getters.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pickle
+from fnmatch import fnmatch
+import boto3
+import json
+
+import pandas as pd
+
+from ai_genomics import bucket_name
+
+#####################################################################
+
+s3 = boto3.resource("s3")
+
+def get_s3_dir_files(s3, bucket_name, dir_name):
+    """
+    get a list of all files in bucket directory.
+    s3: S3 boto3 resource
+    bucket_name: The S3 bucket name
+    dir_name: bucket directory name
+    """
```

Comment:
We should use either short docstrings with no args when a function is very simple or full docstrings with args and returns. Best to use type hints where possible too (also in consistency with the rest of the codebase).

```suggestion
def get_s3_dir_files(
    s3: boto3.resources.base.ServiceResource,
    bucket_name: str,
    dir_name: str
) -> List[str]:
    """
    Get a list of all files in bucket directory.
    
    Args:
        s3: S3 resource
        bucket_name: S3 bucket name
        dir_name: S3 bucket directory name

    Returns:
        list_dir: List of file names in bucket directory
    """
```

You can also use [forward references](https://peps.python.org/pep-0484/#forward-references) but referring to the object type as a string if it hasn't been defined yet.